lib/Sema/SemaExpr.cpp

```diff
@@ -394,11 +394,18 @@ bool Sema::DiagnoseUseOfDecl(NamedDecl *D, SourceLocation Loc,
   // type in checked block.
   // Otherwise, it does not emit redundant error message at the use of variable
   // since it already produced an error message at the declaration of variable.
+  // Calls to variable argument functions are not allowed in a checked block.
```

Comment:
I opened a Checked C spec issue about constructed types that use unchecked pointer types: https://github.com/Microsoft/checkedc/issues/142.   Feel free to comment on it there.